src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerGeneralEmail.java

```diff
@@ -60,6 +60,9 @@ public class CaseworkerGeneralEmail implements CCDConfig<CaseData, State, UserRo
     private static final String NO_VALID_EMAIL_ERROR
         = "You cannot send an email because no email address has been provided for this party.";
 
+    private static final String WARNING_ATTACHMENT_SIZE = "### NOTE: Individual attachments must be less than 2MB"
```

Comment:
Nitpicky suggestion - would "WARNING" catch their attention more than  "NOTE"?
```suggestion
    private static final String WARNING_ATTACHMENT_SIZE = "### WARNING: Individual attachments must be less than 2MB"
```